kahuna/public/js/search/results.js

```diff
@@ -465,7 +468,7 @@ results.controller('SearchResultsCtrl', [
             return $stateParams.query || '*';
         }
 
-        function search({query, until, since, offset, length, orderBy, countAll} = {}) {
+        function search({query, until, since, offset, length, orderBy, useAISearch, countAll} = {}) {
```

Comment:
We probably don't need `useAISearch` in these params as this won't change as we scroll. It loads this from `$stateParams` in the body of the function